lib/cfme/cloud_services/inventory_sync.rb

```diff
@@ -29,22 +29,24 @@ def self.sync_queue(userid, targets)
         MiqTask.generic_action_with_callback(task_opts, queue_opts)
       end
 
-      def self.bundle(manifest:, targets:, tempdir: nil, task_id: nil, miq_task_id: nil)
-        path = DataPackager.package(DataCollector.collect(manifest, targets_from_queue(targets)), tempdir)
+      def self.bundle(manifest:, targets:, tempdir: nil, perm: nil, task_id: nil, miq_task_id: nil)
```

Comment:
maybe remove miq_task_id ?  not used here and will appease rubocop and codeclimate.

Reply:
Can't, MIQ core passes both miq_task_id and task_id.  It is something I plan on refactoring in the future but for now any method that takes kwargs and is called by MiqTask.generic_action_with_callback must accept both.

Reply:
👍 follow-up then.